src/Nancy/GlobalizationConfiguration.cs

```diff
@@ -15,15 +15,22 @@ public class GlobalizationConfiguration
         /// <summary>
         /// Initializes a new instance of the <see cref="GlobalizationConfiguration"/> class
         /// </summary>
-        /// <param name="supportedCultureNames"></param>
-        public GlobalizationConfiguration(IEnumerable<string> supportedCultureNames)
+        /// <param name="supportedCultureNames">An array of supported cultures</param>
+        /// <param name="defaultCulture">The default culture of the application</param>

```

Comment:
Currently if the list is not populated but the default is or the default isn't in the list, the convention  will fail and return null and therefore will fallback to the thread convention. We can add checks in this class and throw an exception if you want though